packages/components/src/LightBox/LightBox.tsx

```diff
@@ -15,15 +15,21 @@ interface PresentedImage {
 interface LightBoxProps {
   readonly open: boolean;
   readonly images: PresentedImage[];
+  readonly imageIndex?: number;
   onRequestClose(): void;
 }
 
 export function LightBox({
   open = false,
   images,
+  imageIndex,
```

Comment:
```suggestion
  imageIndex = 0,
```
I think if you just set a default here you we can skip the conditional in 28-31 and just `const [currentImageIndex, setCurrentImageIndex] = useState(imageIndex);` on line 32

Reply:
Oh yep, totally missed that!